assets/emojis.js

```diff
@@ -1942,6 +1942,7 @@ const emojis = [
             'call_me_hand',
             'call',
             'hand',
+            'shaka',
```

Comment:
lol I would say we should make the `name: 'shaka'` and let the call_me_had be an alias

Reply:
Thought about that actually, but then we'd be deviating from the [UTC standard](https://www.unicode.org/emoji/charts/emoji-list.html#1f919) (shaka is listed as a keyword though which is great to see)

Reply:
lame UTC lame!

Reply:
Call Me Hand... if you call me hand im gonna call you fakka

Reply:
😂